src/Workspaces/Remote/ServiceHub/Host/SolutionAssetCache.cs

```diff
@@ -52,7 +52,7 @@ static SolutionAssetCache()
     /// </summary>
     private readonly TimeSpan _gcAfterTimeSpan;
 
-    private readonly ConcurrentDictionary<Checksum, Entry> _assets = new(concurrencyLevel: 4, capacity: 10);
+    private readonly ConcurrentDictionary<Checksum, Entry> _assets = new();
```

Comment:
We're on .net core.  we don't want or need to artificially limit this dictionary.  esp. when we want to do all this concurrent work.